scout/server/links.py

```diff
@@ -100,6 +101,13 @@ def gnomad_str_gene(hgnc_symbol):
     return link.format(hgnc_symbol)
 
 
+def ckb_gene(entrez_id):
```

Comment:
One could consider a ckb_variant as well, but its based on like protein positions on an reference protein product given on the site... Im sure they are pretty stable for the core set of genes, but still. Not super solid practice. Gene only is a good call until someone says they are would actually really want it.

Reply:
Agreed!